app/policies/comment_policy.rb

```diff
@@ -0,0 +1,31 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+  
+    def initialize(user, comment)  
+      @user = user
+      @comment = comment
+      
+    end
+
+    
+
+    def edit?
+      user == comment.author
+    
+    end
+
+    def update?
+      edit?
+    end
+
+    def destroy?
+      user == comment.author ||
+      user == comment.owner
```

Comment:
`.owner` is not a valid method. I think because of this I'm able to delete anyone's comment